### DISCUSSION TITLE
[Bug] Monarch Tokenizer negative "lookbehind, lookahead" not working

### DISCUSSION THREAD

**Discussion by fdrag** [microsoft/monaco-editor#3684] [Q&A]
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Code

```typescript
return {
  tokenizer: {
    root: [
      [/\b(?<=var)\s\w+/, 'keyword']
    ]
  },
};
```

### Actual Behavior

![image](https://user-images.githubusercontent.com/97387068/148797374-58293997-d688-4684-a73f-669072004a2a.png)

lookahead not woring

### Expected Behavior

![image](https://user-images.githubusercontent.com/97387068/148797434-afa408a8-c353-4bd4-9c91-7f0844718973.png)

### Additional Context

highlight words that doesnt start with var + whitespace

**Comment by 14glwu** [comment]
the same problem

**Comment by medihack** [comment]
I guess this depends on browser support of lookbehind which is not perfect https://caniuse.com/js-regexp-lookbehind

**Comment by jogibear9988** [comment]
As it is created as a feature request, it seems not to be a browser issue: https://github.com/microsoft/monaco-editor/issues/3441
